src/MSALWrapper/TokenResultExtensions.cs

```diff
@@ -9,15 +9,14 @@ namespace Microsoft.Authentication.MSALWrapper.AuthFlow
     internal static class TokenResultExtensions
     {
         /// <summary>
-        /// Sets the AuthType property to the given authType if the token result is not null.
+        /// Sets the Silent property to true if the token result is not null.
         /// </summary>
         /// <param name="result">The result.</param>
-        /// <param name="authType">The auth type.</param>
-        internal static void SetAuthenticationType(this TokenResult result, AuthType authType)
+        internal static void SetSilent(this TokenResult result)
         {
             if (result != null)
             {
-                result.AuthType = authType;
+                result.Silent = true;
             }

```

Comment:
Super confused here, why is `result != null` means token is silent? You could have a non-null result with web as well which is interactive.

Reply:
This method is only used to set the tokenResult's `Silent` property to true. The default value of that property is false. 
This method is called from each authflow when we fetch the token silently. A non-null tokenResult will always be interactive by default.

Reply:
This is a replacement for the method - `SetAuthenticationStatus(result, AuthType)`. But in this PR, the `AuthType`s are removed. It is replaced by a boolean `Silent` which represents whether the token is fetched silently or not.

Reply:
Okay! I saw it now. You are only using it when we get token using `AcquireTokenSilent` or IWA. I have been scrolling up and down to understand the code. A lot of new stuff in CommandMain is implemented below. Is there a reason why we did not add this specific work of change AuthType to Silent or not as a PR in itsself?

Reply:
Yea this was a part of the current feature. I thought of removing the AuthType in a different PR but I got feedback about it on this PR itself so I made changes here.